todo.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+class Todo {
+  constructor() {
+    this.listTodo = [];
+    this.id = 0;
```

Comment:
will be better to name lastId or nextId, because it is not the id of class instance